common/src/main/java/com/revenuecat/purchases/common/verification/SignatureVerificationMode.kt

```diff
@@ -0,0 +1,32 @@
+package com.revenuecat.purchases.common.verification
+
+import com.revenuecat.purchases.EntitlementVerificationMode
+
+sealed class SignatureVerificationMode {
```

Comment:
Now this will be used internally and contain the signature verifier to use to verify signatures. It's a similar pattern to what we have in iOS.